coreloop/commanding.c

```diff
@@ -36,6 +36,8 @@ bool process_cdi(struct core_state* state)
     cdi_fill_command_buffer();
     #endif
 
+void spec_enable_watchdogs(uint8_t enable);
```

Comment:
This stub should be defined in the `src/spectrometer_interface.c`. Anything starting with spec_ is defined there (or in the right place in `luSEE_MiV` for the real hardware)